mapeditor/inspector/inspector.cpp

```diff
@@ -913,6 +909,49 @@ QTableWidgetItem * Inspector::addProperty(CGCreature::Character value)
 
 Inspector::Inspector(MapController & c, CGObjectInstance * o, QTableWidget * t): obj(o), table(t), controller(c)
 {
+	reusedTr = {
```

Comment:
There is generally no need to perform manual deduplication - lupdates automatically groups identical strings

Reply:
Wasn't aware of that. Thanks!

Changed it! :)

Reply:
Yeah. Saw this multiple times when resolving conflicts in .ts files :)

Works only within same 'context' (e.g. in the same class). For same translation in different files lupdate uses different mechanism - existing translation will be copied to a new location, but will be marked as 'unfinished' for validation by translator.